store/views.py

```diff
@@ -54,15 +99,26 @@ def viewLoanedBooks(request):
 @csrf_exempt
 @login_required
 def loanBookView(request):
+    book_id = request.POST.get("bid")
+    book1=Book.objects.get(pk=book_id)
```

Comment:
This may fail with invalid book ID given in POST request, and would lead to server error. Expected behavior is to inform user with Not found (404) error.